src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js

```diff
@@ -8,35 +8,53 @@ import Paper from "@material-ui/core/Paper";
 import Typography from "@material-ui/core/Typography";
 import UnconfirmIcon from "@material-ui/icons/Cancel";
 import Moment from "react-moment";
+import moment from "moment";
 
 import gql from "graphql-tag";
 import { useQuery } from "@apollo/react-hooks";
+import Loading from "components/OmouComponents/Loading";
+import { DayAbbreviation } from "utils";
 
 export const GET_INSTRUCTOR_ENROLLMENTS = gql`
   query InstructorEnrollments($instructorID: ID!) { 
     courses(instructorId: $instructorID) {
       id
-      dayOfWeek
       description
       startDate
-      startTime
       endDate
-      endTime
       isConfirmed
       title
+      availabilityList {
+        id
+        dayOfWeek
+        endTime
+        startTime
+      }
     }
   }
 `;
 
+const checkTimes = (courseTimes) => {
```

Comment:
What are you checking for? rename to clarify

Reply:
This seems like a weird check - you're checking to see if there are any other start/end times?

Reply:
This check is to see if all times for the course are the same, like Wednesday at 4-5 and Thursday at 4-5. If so then the times will be displayed in the table. If they are different then the word "Various" will be displayed in the table instead. I can change the name to clarify the names